unit_tests/tests/test_block_hash_and_number.rs

```diff
@@ -1,22 +1,16 @@
-#![feature(assert_matches)]
+mod common;
+use common::*;
 
-use rpc_test::test_config::TestConfig;
-use starknet::{providers::{
-    jsonrpc::{HttpTransport, JsonRpcClient},
-    Provider,
-}, core::types::BlockId};
+use std::collections::HashMap;
+
+use starknet::{providers::{jsonrpc::{HttpTransport, JsonRpcClient}, Provider}, core::types::BlockId};
```

Comment:
rename into starknet_providers

Reply:
Split imports in commit 56888676622c053e9995b56970f9b7e4542f5e53